Sources/Collections/TableViews/SkeletonTableViewProtocols.swift

```diff
@@ -12,6 +12,7 @@ public protocol SkeletonTableViewDataSource: UITableViewDataSource {
     func numSections(in collectionSkeletonView: UITableView) -> Int
     func collectionSkeletonView(_ skeletonView: UITableView, numberOfRowsInSection section: Int) -> Int
     func collectionSkeletonView(_ skeletonView: UITableView, cellIdentifierForRowAt indexPath: IndexPath) -> ReusableCellIdentifier
+    func collectionSkeletonViewConfigureCell(_ cell: UITableViewCell)
```

Comment:
To follow the same code style I think this signature is more accurate:
```swift
func collectionSkeletonView(_ skeletonView: UITableView, prepareCellForSkeleton cell: UITableViewCell) 
```

WDYT?

Reply:
And for consistency I would also add it to CollectionView's data source

Reply:
exactly, I said the same in the comment above :)

Reply:
Sure thing!